homework_1/task1/fizzbuzz/fizzBuzz.go

```diff
@@ -0,0 +1,40 @@
+package fizzbuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func FizzBuzz(start, end int) ([]string, error){
+	if start > end {
+		return nil, errors.New("range start is greater than range end")
+	}
+
+	if start <= 0 {
+		return nil, errors.New("start is 0 or negative")
+	}
+
+	if end <= 0 {
+		return nil, errors.New("end is 0 or negative")
+	}
+
+	var changedOutput []string
+
+	for i := start; i <= end; i++ {
+		switch{
+		//using mod 15 can save one check instead of (mod 3 && mod 5)
```

Comment:
Should be a whitespace after `//`, but formatting should resolve that issue as well.

Reply:
`go fmt ./...`  actually didn't. Maybe i have to tweak it somewhere in Settings. But I've done it by hand.